packages/boot/tools/drivers.ts

```diff
@@ -281,6 +281,17 @@ export const makeGovernanceDriver = async (
         proposal: {},
       });
     },
+    getOracleInvitation: async () => {
```

Comment:
we have [method naming conventions](https://docs.agoric.com/guides/ertp/#method-naming-structure) and "get" usually throws when it doesn't find the thing... at least in the store API... so consider...

```suggestion
    findOracleInvitation: async () => {
```